cuda_core/tests/test_event.py

```diff
@@ -6,34 +6,46 @@
 # this software and related documentation outside the terms of the EULA
 # is strictly prohibited.
 
-from cuda import cuda
-from cuda.core.experimental._event import  EventOptions, Event
-from cuda.core.experimental._utils import handle_return
+from cuda.core.experimental._event import  EventOptions
 from cuda.core.experimental._device import Device
-import pytest
 
-def test_is_timing_disabled():
+def test_is_timing_disabled(init_cuda):

```

Comment:
Let's call it `test_timing` and parametrize this test with 3 `timing` options: `None`, `True`, `False`. Specialize a case for `None`: We need to test the default (without any `EventOptions`) is no timing (IIRC).